cohere/client.py

```diff
@@ -145,15 +145,21 @@ def generate(self,
         response = self._executor.submit(self.__request, cohere.GENERATE_URL, json=json_body)
         return Generations(return_likelihoods=return_likelihoods, _future=response, client=self)
 
-    def chat(self, query: str, session_id: str = "", persona: str = "cohere", model: str = None) -> Chat:
+    def chat(self,
```

Comment:
can we please add a docstring (similar to the one in summarize?)

Reply:
Yep! On it